lib/compiler/src/beam_trim.erl

```diff
@@ -24,12 +24,12 @@
 -moduledoc false.
 -export([module/2]).
 
--import(lists, [any/2,reverse/1,reverse/2,seq/2,sort/1]).
+-import(lists, [any/2,merge/1,reverse/1,reverse/2,seq/2,sort/1]).
 
 -include("beam_asm.hrl").
 
 -record(st,
-        {safe :: sets:set(beam_asm:label()),    %Safe labels.
+        {safe :: gb_sets:set(beam_asm:label()),    %Safe labels.
```

Comment:
Changing from `sets` to `gb_sets` is unnecessary since the only operation performed on this set is checking for membership (in `is_safe_branch/2`). Generally, checking for membership is faster for `sets` than for `gb_sets`.